src/server/views/amsapp-stadspas-administratienummer.pug

```diff
@@ -1,20 +1,14 @@
 doctype html
 html(lang="en")
   head
-    title Stadspas clientnummer
```

Comment:
hoe kan ik deze file testen?

Reply:
Vertel ik je ff via slack zo.

Reply:
Eigenlijk, denk ik dat we helemaal geen pug hoeven te renderen.